pages/index.jsx

```diff
@@ -1,7 +1,8 @@
 // import { Routes } from "@config/routes";
 import styles from "./index.module.scss";
-import { Header } from "../features/layout/header/index";
-import { Modal } from "../features/layout/modal/index";
+import { Header } from "../features/landing/components/header";
+import { Modal } from "../features/landing/components/modal";
+import { Hero } from "../features/landing/components/hero";
 import { useState } from "react";
 
 const IssuesPage = () => {
```

Comment:
This is a good chances to try out Static Site Generation (SSG) with Next.js using the `getStaticProps` function. This way the whole page is pre-rendered during build time. When a user visits the page they receive a ready-made HTML file which is a lot faster than first serving the skeleton of this page and then requesting the data on the client. 

I'd recommend checking out the Next.js docs about SSG and the `getStaticProps` function